Docs: Update Example Generation Setup

kalib-brayer opened the pull request:
Added bash files to make example loading and updating easier. Now the user can call on a bash file to load or update data instead of manually updating a bash script each time. I also updated the data loading scripts to reference the new bash files.

LukaFontanilla reviewed (commented):
@kalib-brayer, so the new update requires the user to hard code the environment variables in the individual bash script files?

If the environment variables are shared across each bash script, I think they should be moved outside of the individual files into a single shared environment file that the developer can fill in the values for one time. Then we load those variables into the environment each time one of the bash scripts is called (providing examples of how to do that in the documentation).

kalib-brayer commented:
@LukaFontanilla I added a single shared environment file so the developer only needs to fill in the values one time. I also added explicit directions on updating the .env file to reference the necessary variables before running the scripts to load the examples. I simplified the bash scripts to reference the single .env file too.
